test/unit/prompts/SlidedownPrompting/SlidedownPrompting.ts

```diff
@@ -93,9 +196,7 @@ test.serial(`on slidedown dismiss with slidedown queue non-empty, show next slid
       OneSignal.on(Slidedown.EVENTS.QUEUED, () => {
         eventCounts.queued+=1;
         // all slidedowns have been enqueued by this point
-        // first slidedown should be showing
         if (eventCounts.queued === 1) {
-            t.is(eventCounts.shown, 1);
             eventsHelper.simulateNativeAllowAfterShown();
             eventsHelper.simulateSlidedownAllow();
```

Comment:
see if both of these are needed

Reply:
it is needed since, we are awaiting on the subscription promise. but the subscription promise can be removed. so will update